addons/crm/models/res_partner.py

```diff
@@ -64,6 +64,10 @@ def action_view_opportunity(self):
         '''
         action = self.env['ir.actions.act_window']._for_xml_id('crm.crm_lead_opportunities')
         action['context'] = {}
+        action['views'] = [
+            (False, 'list'), (False, 'form'), (False, 'kanban'), (False, 'graph'),
+            (False, 'pivot'), (False, 'calendar'), (False, 'cohort'), (False, 'activity')
```

Comment:
If only CRM is installed, wouldn't the cohort view coming from `crm_enterprise` cause an issue? Could you please check?